module-domain/src/main/kotlin/site/yourevents/invitation/domain/Invitation.kt

```diff
@@ -7,8 +7,14 @@ class Invitation(
     val id: UUID,
     val member: Member,
     var qrUrl: String,
+    var deleted: Boolean
+
 ) {
     fun updateQrCode(qrUrl: String) {
         this.qrUrl = qrUrl
     }
+
+    fun updateDeleted(){
+        this.deleted = true
+    }
```

Comment:
해당 메서드는 초대장을 삭제 처리하는 용도로만 사용되니, 행위를 보다 명확히 명시하면 좋을 것 같아요..!

```suggestion
    fun markAsDeleted(){
        this.deleted = true
    }
```

Reply:
수정하였습니다!!